tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Telemetry.spec.ts

```diff
@@ -0,0 +1,34 @@
+import {test} from '@umbraco/playwright-testhelpers';
+import {expect} from "@playwright/test";
+
+test.describe('Telemetry tests', () => {
+
+  test.beforeEach(async ({page, umbracoApi}, testInfo) => {
+    await umbracoApi.telemetry.setTelemetryLevel("Basic");
+  });
+
+  test.afterEach(async ({page, umbracoApi}, testInfo) => {
+    await umbracoApi.telemetry.setTelemetryLevel("Basic");
+  });
+
+  test('can change telemetry level', async ({page, umbracoApi, umbracoUi}) => {
+    const expectedLevel = "Minimal";
+
+    await page.goto(umbracoApi.baseUrl + '/umbraco');
+
+    await page.pause();
```

Comment:
Don't think we need a page.pause() here 🤣

Reply:
Whoops, fixed now

Reply:
Still not fixed 🤣